src/Core/JVM/Invoker/Extended/PHPMethodAnnotationAffectable.php

```diff
@@ -0,0 +1,30 @@
+<?php
+namespace PHPJava\Core\JVM\Invoker\Extended;
+
+trait PHPMethodAnnotationAffectable
+{
+    public function getAnnotations(string $name): array
```

Comment:
```php
/**
 * @return \phpDocumentor\Reflection\DocBlock\Tag[][]
 */
```

Reply:
That could not be solved method names on my editor.